k8s/client.go

```diff
@@ -459,20 +460,37 @@ func (client *Client) FindDeploymentByLabel(ctx context.Context, label string) (
 
 func (client *Client) DeployCoreOperatorSync(ctx context.Context, coreInstance cloud.CreatedCoreInstance, version string) (*appsv1.Deployment, error) {
 	labels := client.LabelsFunc()
-	const toCloudImage = "ghcr.io/calyptia/core-operator/sync-to-cloud"
-	const fromCloudImage = "ghcr.io/calyptia/core-operator/sync-from-cloud"
-	args := []string{"-cloud_url", client.CloudBaseURL, "-token", client.ProjectToken, "-core_instance", coreInstance.Name, "-namespace", client.Namespace}
+	const toCloudImage = "ghcr.io/calyptia/core-operator/sync-to-cloud:v1.0.0-alpha0"
+	const fromCloudImage = "ghcr.io/calyptia/core-operator/sync-from-cloud:v1.0.0-alpha0"
+	env := []apiv1.EnvVar{
```

Comment:
if value isn't provided don't add it to envvar

Reply:
none of those values are not provided, token, url, and namespace are defaults and core instance name fails before if we dont have it.

Reply:
but i can include a check if we think that is needed

Reply:
please do otherwise we might end up with a failing setup